src/backend/metrics/metrics.ts

```diff
@@ -268,8 +275,9 @@ export async function trackPidPlaytime(
           isBrowserGame: false,
           game_name: gameInfo.app_name,
           game_title: gameInfo.title,
-          store_name: gameInfo.runner,
+          store_name: getStoreName(gameInfo.runner),
           processName: processInfo.name,
+          platform: getPlatformName(gameInfo.install.platform!),
```

Comment:
same here. It won't send now `windows_amd64` only Windows, Linux, macOS or Browser.
Should I keep this change? @awantoch

Reply:
as a sidenote, I think we should condense all of our platform strings on game info into shared ones across all game stores (separate PR). Right now it's a leaky abstraction with different store naming conventions propagating everywhere

We should define a common interface for stores and enforce all stores to meet this standard. I think we probably only need "Windows, Linux, macOS or Browser" and the _amd64 etc can be inferred by the current arch

here's an issue I made for this https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/issues/627

Reply:
ah yeah that makes sense -- Windows, Linux, macOS, Browser, but still probably helpful to include the architecture on the dev portal side, since we'll need to determine which version we run (mainly on macOS, determining whether to run the arm or amd version)

I still think it would be helpful to include the full platform_architecture combo in the log though for data purposes, if not just to report back to the developers 🤔

Reply:
@awantoch so I keep as it is or use both then? we can have both, it is fine.

Reply:
Added `platform_arch` to be used if we want to refine the architecture like you said 👍🏽